### ISSUE TITLE
vault hangs when attempting to unseal

### ISSUE THREAD

**Issue by jaysiyani** [hashicorp/vault#10184]
**Describe the bug**

We have vault running in HA with a unseal service that runs every 30 seconds. This unseal service runs on all 3 vault instances (1 master 2 standby).
Recently we've had an issue where once the master has been unsealed, the unseal process fails (even when ran manually) and causes vault to become unresponsive.
The unseal process still works on the other 2 standby nodes. It only seems to affect the master. 

This unseal process has been running without any issues for at least 4 days using vault 1.4.6, previously it was running fine on vault 1.4.2 for over 4 months. 

At the time of failure there was also a higher than normal number of Goroutines. 

**Expected behaviour**

The unseal process should attempt to unseal vault and if it's already unsealed it should just exit. 

**Environment:**

- Flatcar Container Linux by Kinvolk stable (2512.4.0)
- Vault docker image, Version: 1.4.6 (also 1.4.2, which we downgraded to in an attempt to resolve this issue)
- Consul docker image, Version: 1.2.2

Vault server configuration file(s):

```hcl
api_addr = "https://vault-xxx.co"
cluster_addr = "https://172.24.x.x:8201"
storage "consul" {
  address = "172.24.x.x:8500"
  path    = "vault"
  service = "vault"
}

telemetry {
    statsd_address = "statsd:9125"
  disable_hostname = true

}

listener "tcp" {
  address = "0.0.0.0:8200"
  tls_key_file = "/etc/ssl/vault-new/vault/key"
  tls_cert_file = "/etc/ssl/vault-new/vault/cert"
  tls_disable_client_certs = true
}

log_level = "Debug"
log_format = "json"
```

**Additional context**

I have attached the stack trace from the Goroutines.

[vault-2020-10-16T14_41_52+00_00.log](https://github.com/hashicorp/vault/files/5409288/vault-2020-10-16T14_41_52%2B00_00.log)

**Comment by aphorise** [comment]
I'm believe that this was already resolved and using the most recent versions (1.11.3) I was unable to reproduce this.
Hey @jaysiyani can you confirm if you've retested or if this is still applicable for you?

**Comment by hsimon-hashicorp** [comment]
I'm going to go ahead and close this issue now, but please feel free to re-open if needed. Note, we accept bug reports for "n-2" versions of Vault, so 1.10+ as of this message. Thanks!
